doc/source/user_guide/style.ipynb

```diff
@@ -991,14 +983,12 @@
    "outputs": [],
    "source": [
     "np.random.seed(25)\n",
-    "cmap = cmap=sns.diverging_palette(5, 250, as_cmap=True)\n",
+    "cmap = cmap = sns.diverging_palette(5, 250, as_cmap=True)\n",
     "bigdf = pd.DataFrame(np.random.randn(20, 25)).cumsum()\n",
     "\n",
-    "bigdf.style.background_gradient(cmap, axis=1)\\\n",
-    "    .set_properties(**{'max-width': '80px', 'font-size': '1pt'})\\\n",
-    "    .set_caption(\"Hover to magnify\")\\\n",
-    "    .set_precision(2)\\\n",
-    "    .set_table_styles(magnify())"
+    "bigdf.style.background_gradient(cmap, axis=1).set_properties(\n",
+    "    **{\"max-width\": \"80px\", \"font-size\": \"1pt\"}\n",
+    ").set_caption(\"Hover to magnify\").set_precision(2).set_table_styles(magnify())"
```

Comment:
For example this one (related to my comment above), I don't directly find it an improvement ..